source/computer/wsl-setup.rst

```diff
@@ -51,6 +53,9 @@ WSL 有 WSL1 和 WSL2 两个发行版本，二者底层原理不同。大多数
 
        $ bcdedit /set hypervisorlaunchtype auto
 
+   新版本 VMware (>=15.5.5) 和 VirtualBox (>=6.0) 在最新 Windows 系统上
+   （build number >= 19041）可以正常使用，无兼容性问题。
```

Comment:
I don't find the build number requirement on the official documentation.

Reply:
It's here (https://blogs.vmware.com/workstation/2020/05/vmware-workstation-now-supports-hyper-v-mode.html).

> To run Workstation/Player using the Windows Hypervisor APIs, the minimum required Windows 10 version is Windows 10 20H1 build 19041.264. VMware Workstation/Player minimum version is 15.5.5.